packages/app/src/admin/InvitePage.tsx

```diff
@@ -24,15 +25,16 @@ export function InvitePage(): JSX.Element {
     <Form
       onSubmit={(formData: Record<string, string>) => {
         const body = {
-          resourceType: formData.resourceType,
+          resourceType: formData.resourceType as 'Practitioner' | 'Patient' | 'RelatedPerson',
```

Comment:
You can use `as ProfileResource`

`import { ProfileResource } from '@medplum/core';`